tools/launchdev.sh

```diff
@@ -209,11 +209,10 @@ function st2start(){
     cp -Rp ./contrib/core/ $PACKS_BASE_DIR
     cp -Rp ./contrib/packs/ $PACKS_BASE_DIR
 
-    if [ "$copy_examples" = true ]; then
-        echo "Copying examples from ./contrib/examples to $PACKS_BASE_DIR"
+    if [ "$copy_test_packs" = true ]; then
+        echo "Copying test packs examples and tests to $PACKS_BASE_DIR"
         cp -Rp ./contrib/examples $PACKS_BASE_DIR
-        cp -Rp ./st2tests/tests $PACKS_BASE_DIR
-        cp -p ./st2tests/tests/configs/tests.yaml $CONFIG_BASE_DIR
+        cp -Rp ./st2tests/packs/tests $PACKS_BASE_DIR
```

Comment:
Please let me know if this is the right path `./st2tests/packs/tests`

Reply:
The `tests` pack is in a completely different repo https://github.com/StackStorm/st2tests/. You have to git clone https://github.com/StackStorm/st2tests.git to a temporary location and then copy the directory `./st2tests/packs/tests/` from the temporary location to /opt/stackstorm/packs.